smac/configspace/util.py

```diff
@@ -49,9 +49,8 @@ def impute_default_values(
         with their default value.
     """
     for hp in configuration_space.get_hyperparameters():
-        default = hp.normalized_default_value
         idx = configuration_space.get_idx_by_hyperparameter_name(hp.name)
         nonfinite_mask = ~np.isfinite(configs_array[:, idx])
-        configs_array[nonfinite_mask, idx] = default
+        configs_array[nonfinite_mask, idx] = -1
```

Comment:
docstring says "imputed with their default value". Is this still true?

Reply:
No, I updated the text.